plugins/services/src/js/components/ServiceStatusIcon.js

```diff
@@ -49,41 +67,45 @@ class ServiceStatusWarning extends Component {
       );
     }
 
-    return <noscript />;
+    return null;
   }
 
   getTooltip(content) {
-    let icon = <Icon color="red" id="yield" size="mini" />;
+    const { service } = this.props;
+    let icon = <StatusIcon state="WARNING" />;
```

Comment:
It would be nice if "WARNING" came from a constants file so it was immediately clear what possible values can be passed into here.

Reply:
This is basically a duplication of:
https://github.com/dcos/dcos-ui/pull/2475#discussion_r148068634
https://github.com/dcos/dcos-ui/pull/2475#discussion_r148768468

Thanks for the reminder though.